minecraft_servers/thelotus/manifest.json

```diff
@@ -0,0 +1,28 @@
+{
+    "server_name": "thelotus",
+    "nice_name": "TheLotus",
+    "direct_ip": "thelotus.tv",
+    "server_wildcards": [
+        "mc.thelotus.tv"
+    ],
+    "supported_languages": [
+        "en",
+        "de"
+    ],
+    "social": {
+        "web": "https://thelotus.tv/",
+        "web_support": "https://thelotus.tv/forum/",
+        "twitter": "TheLotusTeam",
+        "tiktok": "TheLotusNetwork",
+        "instagram": "LotusServer",
+        "discord": "https://lts.link/discord",
```

Comment:
Please use the direct invite discord.gg/invite/...
[Documentation reference](https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#discord-url)